core-bundle/contao/languages/en/tl_content.xlf

```diff
@@ -800,6 +800,12 @@
       <trans-unit id="tl_content.tableLabel">
         <source>Content element</source>
       </trans-unit>
+      <trans-unit id="tl_content.serverPagination.0">
+        <source>Server-side pagination</source>
+      </trans-unit>
+      <trans-unit id="tl_content.serverPagination.1">
+        <source>Enables server-side pagination, rather than client side.</source>
```

Comment:
How helpful is this for a regular user? Do they know what the difference is or when they should enable this option?

Reply:
> How helpful is this for a regular user?

Do you mean the phrasing or the feature itself?

Reply:
How about something like

> Switches from a pagination generated by JavaScript to one generated by the server.

Or

> Makes the pagination accessible by query parameters and only renders the elements for a specific page.